x-pack/legacy/plugins/beats_management/common/constants/index.ts

```diff
@@ -9,4 +9,4 @@ export { INDEX_NAMES } from './index_names';
 export { PLUGIN } from './plugin';
 export { LICENSES, REQUIRED_LICENSES, REQUIRED_ROLES } from './security';
 export { TABLE_CONFIG } from './table';
-export const BASE_PATH = '/management/beats_management';
+export const BASE_PATH = '/management/beats/beats_management';
```

Comment:
The way the new Management plugin works, this is unavoidable. This PR will change the URL for this UI, however since it's marked "beta" this should be fine.